### DISCUSSION TITLE
Cannot build in Gnome Builder

### DISCUSSION THREAD

**Discussion by UserrX** [GradienceTeam/Gradience#765] [Q&A] [answered]
When I try to build per the instructions I get the following error in Builder

`../../../../../../Projects/Gradience/meson.build:46:0: ERROR: Dependency "pygobject-3.0" not found, tried pkgconfig and cmake`

Output of pacman -Ss pygobjects is as follows:

```
extra/python-gobject 3.44.1-3 [installed]
    Python Bindings for GLib/GObject/GIO/GTK+
```

I'm not sure what is going wrong. Any help appreciated.

**Comment by UserrX** [comment]
Ok... I think I fixed the above problem by running 'Update Dependencies' from Gnome Builder. It looks like everything built all right. Now, when I try to run the flatpak package from installer (Arch), I get the following error:

`The application com.github.GradienceTeam.Gradience.Devel/x86_64/master requires the runtime org.gnome.Platform/x86_64/master which was not found`

**Reply by 0xMRTT** [reply, marked as answer]
Add gnome nighty flatpak repo and install the missing runtime

**Reply by UserrX** [reply]
> nome nighty flatpak

Thanks for the reply. Can you tell me how to do this on Archlinux, please?

**Reply by daudix** [reply]
https://wiki.gnome.org/Apps/Nightly

**Reply by 0xMRTT** [reply]
Install flatpak using pacman if it's not installed and run 

`flatpak remote-add --if-not-exists gnome-nightly https://nightly.gnome.org/gnome-nightly.flatpakrepo`

**Reply by UserrX** [reply]
> Install flatpak using pacman if it's not installed and run
> 
> `flatpak remote-add --if-not-exists gnome-nightly https://nightly.gnome.org/gnome-nightly.flatpakrepo`

Thanks for the help, both of you. That worked for me. However, I can now find Gradience in my application menu, but it won't start. It loading for a bit and the disappears.

**Reply by 0xMRTT** [reply]
flatpak run com.github.GradienceTeam.Gradience ?

**Reply by UserrX** [reply]
> flatpak run com.github.GradienceTeam.Gradience ?

`error: app/com.github.GradienceTeam.Gradience/x86_64/master not installed`

I've used Builder all the way and including creating flatpak bundle (though I have no idea what I'm doing). At the end a Nautilus instance opens which shows me the .flatpak file.

Thank you for your patience...
